src/safecast.yaml

```diff
@@ -0,0 +1,31 @@
+---
+description: >
+  Safecast is a global volunter-centered **Citizen Science** project working to
+  empower people with open data about their environments.
+license: MIT
+name: Safecast
+programmingLanguages:
+  - C++
+  - C
+  - JavaScript
+  - PHP
+  - Go
```

Comment:
All languages have to be [lowercase](https://github.com/awesome-global-contributions/awesome-global-contributions/blob/master/CONTRIBUTING.md#necessary-information)

So it should be:

```md
  - c++
  - c
  - javascript
  - php
  - go
```